wgengine/netstack/netstack.go

```diff
@@ -913,13 +921,19 @@ func (ns *Impl) acceptTCP(r *tcp.ForwarderRequest) {
 		}
 	}
 
-	if ns.ForwardTCPIn != nil {
-		c := createConn()
-		if c == nil {
-			return
+	if ns.GetTCPHandlerForFlow != nil {
+		handler, ok := ns.GetTCPHandlerForFlow(clientRemoteAddrPort, dstAddrPort)
+		if ok {
+			if handler == nil {
+				r.Complete(true)
```

Comment:
You complete(true) to refuse a connection? Weird :)

Reply:
This is extremely confusing 😓 I remember tripping over this when working on netstack previously 😵‍💫 The boolean parameter is at least named `sendReset`, I guess?

Reply:
yep
```
func (r *ForwarderRequest) Complete(sendReset bool) {
```